docs/changelog.md

```diff
@@ -133,6 +133,10 @@ permalink: /changelog/
 * **browser-session**
   * ⚠️ **This is a breaking change**: Added `url` parameter to `Session.Observer.onLoadRequest()`.
 
+* **feature-prompts**
```

Comment:
@NotWoods I think we need to move this entry to `2.0.0` section, and will be ready to go  :)